stripe/src/main/java/com/stripe/android/view/PaymentAuthWebViewClient.kt

```
@@ -42,7 +42,7 @@ internal class PaymentAuthWebViewClient(
             onAuthCompleted(
                 runCatching {
                     Uri.parse(url)
-                }.getOrNull()
```

are we intentionally swallow the exception here?

---

Good catch, my change is incorrect.